docs/articles/nunit/release-notes/framework.md

```diff
@@ -2,9 +2,151 @@
 uid: frameworkreleasenotes
 ---
 <!-- markdownlint-disable-file MD013 -->
+<!-- markdownlint-disable-file MD033 -->
 
 # Framework Release
 
+## NUnit 4.3.0 -  ETA Dec 15. 2024
+
+This release includes a series of enhancements and bug fixes. Notably, some of the bug fixes address issues related to a lack of type safety in NUnit. See issues [4877](https://github.com/nunit/nunit/issues/4877),
+[4876](https://github.com/nunit/nunit/issues/4876), [4875](https://github.com/nunit/nunit/issues/4875), [4874](https://github.com/nunit/nunit/issues/4874) and [150](https://github.com/nunit/nunit/issues/150).  
+
+Be aware that this might cause compile errors in cases where the code was previously ignored. These errors can be safely resolved by removing the affected code..
+
+There are 27 issues fixed in this release.
+
+### Enhancements
+
+* [4890](https://github.com/nunit/nunit/issues/4890) Add .net 8 as a target framework in the package. Thanks to NUnit Team member [Manfred Brands](https://github.com/manfred-brands) for [PR 4884]( https://github.com/nunit/nunit/pull/4884)
+* [4886](https://github.com/nunit/nunit/issues/4886) Optimize ToArray()[0] with First() in TestAssert class in TestAssert.cs. Thanks to [](https://github.com/Saurav-shres) for [PR 4888](https://github.com/nunit/nunit/pull/4888)
```

Comment:
`Thanks to []` will not show anything in the resulting markdown

Reply:
Hmm, something amiss there, if no name it should put the account id there instead.

Reply:
Fixed